README.md

```diff
@@ -294,37 +294,66 @@ of that property field.
 ### @fold
 
 Applying `@fold` on a scope "folds" all outputs from within that scope: rather than appearing
-on separate rows in the query result, the folded outputs are coalesced into lists starting
+on separate rows in the query result, the folded outputs are coalesced into parallel lists starting
 at the scope marked `@fold`.
 
+It is also possible to output or apply filters to the number of results captured in a `@fold`.
+The `_x_count` meta field that is available within `@fold` scopes represents the number of elements
+in the fold, and may be filtered or output as usual. As `_x_count` represents a count of elements,
+marking it `@output` will produce an integer value. See the [\_x_count](#_x_count) section for
+more details.
+
 #### Example Use
 ```graphql
 {
     Animal {
         name @output(out_name: "animal_name")
         out_Animal_ParentOf @fold {
+            _x_count @output(out_name: "child_count")
```

Comment:
It may be nice to include type coercion here since it is listed explicitly as allowed

Reply:
Good call.